include/swift/ABI/Metadata.h

```diff
@@ -308,7 +308,7 @@ struct TargetMetadata {
   }
 
   const uint8_t *getLayoutString() const {
-    assert(hasLayoutString());
+    //assert(hasLayoutString());
```

Comment:
Need to make sure that the flag is properly set for all types now. I'll re-enable this before merging.